R/geom-dotplot.r

```diff
@@ -18,16 +18,19 @@
 #'
 #' @eval rd_aesthetics("geom", "dotplot")
 #' @section Computed variables:
+#' These are calculated by the 'stat' part of layers and can be accessed with
+#' [delayed evaluation][aes_eval].
 #' \describe{
```

Comment:
Should we just switch these to regular bulleted lists? Is the output of a `\describe` that much better?

Reply:
I think there are downsides to both `\describe` and to `\itemize`. `\describe` is slightly too spacious for my taste, whereas `\itemize` looks messy because the description follows the item name without any visual cue that they are separate. I think to have best of both words, we could use `\itemize` with `\cr` to separate the item name from description, which would then look like this:

* `after_stat(density)`
  density estimate

* `after_stat(count)`
  density * number of points - useful for stacked density plots.